ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -157,6 +176,8 @@ internal class PaywallViewModelImpl(
             } catch (e: PurchasesException) {
                 listener?.onPurchaseError(e.error)
             }
+
+            _actionInProgress.value = false
```

Comment:
Hmm I guess depending on the implementation, the paywall might be dismissed by the time we execute this, but I think it should be ok even if that's the case.

Reply:
Yeah 👍🏻